images/goland/Dockerfile.ubuntu

```diff
@@ -1,24 +1,7 @@
-FROM ubuntu:20.04
+FROM codercom/enterprise-multieditor:ubuntu
 
-RUN apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get install -y \
-    git \
-    bash \
-    curl \
-    htop \
-    man \
-    vim \
-    ssh \
-    sudo \
-    lsb-release \
-    ca-certificates \
-    locales \
-    gnupg \
-    # Packages required for multi-editor support
-    libxtst6 \
```

Comment:
do we need these for the multi-editor support?

Reply:
Yep, but they're all in `codercom/enterprise-multieditor` now instead of duplicated in all images.